app/assets/javascripts/tree.js

```diff
@@ -216,6 +205,25 @@ Tree.prototype.updateVisibleTreeitems = function () {
   }
 };
 
+Tree.prototype.updateFileCount = function () {
+  var count = this.fileTreeitems.filter(fileTreeItem => fileTreeItem.isSearchVisible).length;
+  var count_text = `${count} ${count == 1 ? "item" : "items"}`;
+  document.querySelectorAll('.sul-embed-item-count').forEach((elem) => elem.innerHTML = count_text);
+};
+
+Tree.prototype.updateSearch = function () {
+  const search = this.searchInput.value.toLowerCase();
+  for (var i = 0; i < this.fileTreeitems.length; i++) {
+    var ti = this.fileTreeitems[i];
+    ti.updateSearch(search);
+  }
+  this.updateFileCount();
+  for (var i = 0; i < this.treeitems.length; i++) {
```

Comment:
consider using `this.fileTreeitems.forEach((ti) => { })`

Reply:
I was just following the W3C example code. It seemed odd to me, but figured they had a reason.